x-pack/plugins/cross_cluster_replication/public/app/components/form_entry_row.js

```diff
@@ -49,7 +49,15 @@ export class FormEntryRow extends PureComponent {
   onFieldChange = (value) => {
     const { field, onValueUpdate, type } = this.props;
     const isNumber = type === 'number';
-    onValueUpdate({ [field]: isNumber ? parseInt(value, 10) : value });
+
+    let valueParsed = value;
+
+    if (isNumber) {
+      valueParsed = !!value ? parseInt(value, 10) : value; // make sure we don't send NaN value
```

Comment:
This would have been better with a `Math.max(0, parseInt(value, 10))` and removing the next line. I'll update it.